docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -3,11 +3,13 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= OpenID Connect (OIDC) and OAuth2 Client and Filters Reference Guide
+= OpenID Connect (OIDC) and OAuth2 client and filters
 include::_attributes.adoc[]
+:diataxis-type: reference
 :categories: security
 :topics: security,oidc,client
 :extensions: io.quarkus:quarkus-oidc-client
+:summary: Use Quarkus extensions to acquire and refresh access tokens from OIDC and OAuth 2.0 compliant servers and propagate access tokens.
 
 This reference guide explains how to use:
 
```

Comment:
@rolfedh: Thanks for considering the changes. Just one comment on the corresponding line 16. @sberyozkin advised that "authorization" be removed from "OIDC and OAuth 2.0 compliant authorization servers" in the newly added summary line. Should this update also be made on line 16 and for that line to be tweaked to the following:

 "- `quarkus-oidc-client`, `quarkus-oidc-client-reactive-filter` and `quarkus-oidc-client-filter` extensions to acquire and refresh access tokens from OIDC and OAuth 2.0 compliant servers such as link:https://www.keycloak.org[Keycloak]"